import-export-cli/Subscription-Policies/AsyncBronze.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This is not related to code I guess. A test artifact that you exported?
There are several policy files like this below. Let's remove all of them.

Reply:
Removed all the artifacts